publishing/submitting/README.md

```diff
@@ -0,0 +1,7 @@
+# Submitting your app to the App Warehouse
+
+To make your app discoverable by others, you're invited to publish it in the App Warehouse.
+
+Currently, the submitting apps to the App Warehouse is invitation-only.
```

Comment:
Currently, the submitting apps to the App Warehouse is invitation-only.  ->
Submitting apps to the Warehouse requires an invition.